jslack-app-backend/src/main/java/com/github/seratch/jslack/app_backend/util/JsonPayloadExtractor.java

```diff
@@ -19,7 +19,7 @@
 
     public String extractIfExists(String requestBody) {
 
-        if (requestBody == null) {
+        if (requestBody == null || requestBody.trim().length() == 0) {
```

Comment:
This one is not related to `auth.test` API updates but it seems like a potential bug.